DocumentsFromSnapshotMigration/src/test/java/com/rfs/SourceTestBase.java

```diff
@@ -80,30 +80,30 @@ protected static void migrateMetadata(
         List<String> indexTemplateAllowlist,
         List<String> indexAllowlist,
         MetadataMigrationTestContext context,
-        ClusterVersion sourceVersion
+        Version sourceVersion
     ) {
         SourceResourceProvider sourceResourceProvider = SourceResourceProviderFactory.getProvider(sourceVersion);
         SnapshotRepo.Provider repoDataProvider = sourceResourceProvider.getSnapshotRepoProvider(sourceRepo);
         GlobalMetadata.Factory metadataFactory = sourceResourceProvider.getGlobalMetadataFactory(repoDataProvider);
+        var targetVersion = Version.fromString("OS 2.11");
```

Comment:
Not for this PR, but once target cluster is also fully dynamically this should be removed.